commcare_connect/multidb/management/commands/setup_logical_replication.py

```diff
@@ -68,53 +79,48 @@ def handle(self, *args, **options):
+                # Create new publication
+                tables = ", ".join([f'"{table}"' for table in table_list])
+                cursor.execute(f"CREATE PUBLICATION {publication_name} FOR TABLE {tables};")
+                self.stdout.write(self.style.SUCCESS(f"Publication '{publication_name}' created successfully."))
 
         # Create subscription in the secondary database
         secondary_conn = connections[secondary_db_alias]
         self.stdout.write("Creating subscription in the secondary database...")
         subscription_name = "tables_for_superset_sub"
 
         with secondary_conn.cursor() as cursor:
-            try:
-                # Check if subscription exists
-                cursor.execute("SELECT subname FROM pg_subscription WHERE subname = %s;", [subscription_name])
-                if cursor.fetchone():
-                    self.stdout.write(
-                        self.style.WARNING(f"Subscription '{subscription_name}' already exists. Skipping creation.")
-                    )
-                else:
-                    # Create new subscription
-                    default_db_settings = default_conn.settings_dict
-                    primary_conn_info = (
-                        f"host={default_db_settings['HOST']} "
-                        f"port={default_db_settings['PORT']} "
-                        f"dbname={default_db_settings['NAME']} "
-                        f"user={default_db_settings['USER']} "
-                        f"password={default_db_settings['PASSWORD']}"
-                    )
-                    cursor.execute(
-                        f"""
-                        CREATE SUBSCRIPTION {subscription_name}
-                        CONNECTION '{primary_conn_info}'
-                        PUBLICATION {publication_name};
-                        """
-                    )
-                    self.stdout.write(self.style.SUCCESS(f"Subscription '{subscription_name}' created successfully."))
-            except Exception as e:
-                raise CommandError(f"Failed to create subscription: {e}")
+            # Check if subscription exists
+            cursor.execute("SELECT subname FROM pg_subscription WHERE subname = %s;", [subscription_name])
+            if cursor.fetchone():
+                self.stdout.write(
+                    self.style.WARNING(f"Subscription '{subscription_name}' already exists. Skipping creation.")
+                )
+            else:
+                # Create new subscription
+                default_db_settings = default_conn.settings_dict
+                primary_conn_info = (
+                    f"host={default_db_settings['HOST']} "
+                    f"port={default_db_settings['PORT']} "
+                    f"dbname={default_db_settings['NAME']} "
+                    f"user={default_db_settings['USER']} "
+                    f"password={default_db_settings['PASSWORD']} "
+                    f"sslmode=disable"  # same VPC in AWS
```

Comment:
why disable SSL, especially if we don't for any of the other actions?

Reply:
This is undone, it was one of the things I tried.